spec.bs

```diff
@@ -670,6 +677,30 @@ button.
 documentPictureInPicture.requestWindow({ disallowReturnToOpener: true });
 </pre>
 
+## Disallow position reuse ## {#example-disallow-position-reuse}
+
+While a document picture-in-picture window is open, the user may choose
+to resize or reposition it. If the document picture-in-picture window is
+closed, then reopened later, the user agent may use the previous position
+and size as a hint for where to place the new window.
+
+The site can provide a hint to the user agent that reusing the previous
+document picture-in-picture window position and size is not desirable
+by setting the {{DocumentPictureInPictureOptions/ignorePreviousWindowBounds}}
+option.  For example, if the site is requesting the new document picture-in-
+picture window for an unrelated activity from the previous one, then the site
+might provide this hint to the user agent.  In response, the user agent may
+choose to use the default position, the default size, or the size hint
+provided by the site instead.
+
+<pre class="lang-javascript">
+documentPictureInPicture.requestWindow({
```

Comment:
May I recommend to either prepend this with `await` or add `.then(...)` so that it's clear `requestWindow` returns a promise?

Reply:
the other example for return-to-tab doesn't contain either of these.  i suspect they should be the same, but i'm not sure if i should change both as part of this change.  @steimelchrome ?

Reply:
Feel free to change as part of this change. Or alternatively open an issue and tell me to do it separately :)

Reply:
done, for both.  i wasn't sure if a change to an otherwise unrelated part of the spec was okay or not.